lightwood/encoders/text/pretrained.py

```diff
@@ -0,0 +1,362 @@
+"""
+2021.03.11: TODO; remove print statements
+Removing the sent_embedder and opting for
+the CLS token only.
+
+2021.03.10
+"CLS" token instead of sent embedder in encode.
+
+2021.03.07
+TODO:
+Freeze base_model in DistilBertModel
+add more complicated layer and then build a model.
+
+Pre-trained embedding model.
+
+This deploys a hugging face transformer
+and trains for a few epochs onto the target.
+
+Once this has been completed, it provides embedding
+using the updated transformer embedding.
+
+NOTE - GPT2 does NOT have a padding token!!
```

Comment:
Is this handled in the PR? If the `lightwood.config` specifies a GPT2 text encoder, what happens to sentences that need to be padded to construct the tensor?

Reply:
I'm going to opt to stick with distilbert since its at least currently faster than the old implementation; I need to look into GPT-2 specifically